legacy/ExportPIM.js

```diff
@@ -3,6 +3,8 @@ var crypto = require('crypto');
 const https = require('https');
 const archiver = require('archiver');
 const ReadableStream = require('stream').Readable;
+const NUMBER_TYPE = 'number';
```

Comment:
perhaps capitalize these as well? similar to the other file change in this PR?

or otherwise lowercase the other one and toLowercase() it instead

Reply:
If we want to standardize then most of the constants in the repo are in uppercase, but having this `number` in ExportPIM.js as uppercase would require an extra step, as `typeof` returns lowercase strings corresponding to the primitive data types. So to have this in uppercase means we need to do `NUMBER_TYPE.toLowerCase()`